examples/WorkstationWriteBasicCookstyle.md

```diff
@@ -0,0 +1,143 @@
+# How-to - Write a basic Cookstyle Rule
```

Comment:
This is a great start! Would it be possible to add a working example with some details about each section of it?

Reply:
It is. The document is going to need to change, anyhow, and add how to do these things in the latest Rubocop versions.

Reply:
done!

Reply:
thanks!